stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py

```diff
@@ -21,7 +21,9 @@ def _compute_product_available_qty(self):
         for location_id, order_in_loc in op_by_loc.items():
             products = order_in_loc.mapped('product_id').with_context(
                 location=location_id.id)._compute_quantities_dict(
-                lot_id=None, owner_id=None, package_id=None)
+                lot_id=self.env.context.get('lot_id'), 
+                owner_id=self.env.context.get('lowner_id'), 
```

Comment:
Typos in variables but I've deleted my comment. I didn't realized that we are in orderpoints so there is no need of this.